pkg/stats/statsmgr.go

```diff
@@ -92,3 +133,16 @@ func (s *StatsMgr) startWorker(numReadingFiles int) {
 		}
 	}
 }
+
+func countLines(path string) int64 {
```

Comment:
This will cause the file to be read twice, will it affect performance?

Reply:
Will the size of the file be considered, not the number of lines in the file? Add `TotalSize`, `ProcessedSize`?